[docs] add debug commands info (#4981)
organicmaps/organicmaps: docs/DEBUG_COMMANDS.md

For more information, please see the source code at [`Framework::ParseSearchQueryCommand`](map/framework.cpp).

## Drape

### Themes

- `?dark` or `mapstyle:dark`: Enable night mode for the map view only. You may need to change the zoom level to fully apply the new mode.
- `?light` or `mapstyle:light`: Enable light mode for the map view only. You may need to change the zoom level to fully apply the new mode.
- `?vdark` or `mapstyle:vdark`: Same as `?dark`, but using the vehicle variant.
- `?vlight` or `mapstyle:vlight`: Same as `?light`, but using the vehicle variant.

### Post processing

- `?aa` or `effect:antialiasing`: Enable antialiasing.
- `?no-aa` or `effect:no-antialiasing`: Disable antialiasing.

### Map layers

- `?scheme`: Enable the subway layer.
- `?no-scheme`: Disable the subway layer.
- `?isolines`: Enable the isolines layer.
- `?no-isolines`: Disable the isolines layer.

### Information

- `?debug-info`: Show renderer version, zoom scale and FPS counter in the top left corner of the map.
- `?debug-info-always`: Same as `?debug-info`, but persists across restarts.

> - `?debug-rect`: TODO

Reviewer: Show debug info and show outlines arround icons and texts on map

- `?no-debug-rect`: TODO.

### Engine

All the following commands require a restart to be applied.

- `?gl`: Enables the OpenGL renderer. Platforms: All.
- `?vulkan`: Enables the Vulkan renderer. Platforms: Android.

> - `?metal`: Enables the Metal renderer. Platforms: TODO.

Reviewer: ```suggestion
- `?metal`: Enables the Metal renderer. Platforms: iOS and macOS.
```

Author: I don't know the Apple ecosystem, do all iphones/Mac support metal?

Reviewer: https://en.wikipedia.org/wiki/Metal_(API)

> ### Engine
> 
> All the following commands require a restart to be applied.
> 
> - `?gl`: Enables the OpenGL renderer. Platforms: All.
> - `?vulkan`: Enables the Vulkan renderer. Platforms: Android.
> - `?metal`: Enables the Metal renderer. Platforms: iOS and macOS.

Reviewer 2: ```suggestion
### Drape rendering engine

All the following commands require app restart:

- `?gl`: Forces the OpenGL renderer. OpenGL renderer is supported on all platforms and is used by default on older Android devices and on the desktop.
- `?vulkan`: Forces the Vulkan renderer on Android. Vulkan is used by default on newer Android 7+ devices.
- `?metal`: Forces the Metal renderer. It is used by default on iOS and can be ported/enabled with some effort on Mac OS X too.
```

## Editor

- `?edits`: Shows the list of edits (local and uploaded) made with the app in the search results.
- `?eclear`: Clears the list of edits. Note: this only clears the app data about edits and does not revert changes in OSM.

## Routing

> - `?debug-cam`: TODO

Reviewer 2: ```suggestion
- `?debug-cam`: Force-enables speed cameras in all countries.
```

> - `?edits`: Shows the list of edits (local and uploaded) made with the app in the search results.
> - `?eclear`: Clears the list of edits. Note: this only clears the app data about edits and does not revert changes in OSM.

Reviewer 2: ```suggestion
- `?edits`: Shows the list of map edits (local and uploaded) made with the app in the search results. Useful to debug what was not uploaded yet.
- `?eclear`: Clears the locally stored list of edits. Warning: all local changes that are not yet uploaded to OpenStreetMap.org will be lost! Everything that was already uploaded to OSM stays there untouched.
```

- `?no-debug-cam`: Reverts speed camera setting to default.

## GPS

> - `?gpstrackaccuracy:XXX`: Changes the accuracy of the gps. Replace `XXX` by the desired horizontal accuracy. TODO needs more info.

Reviewer: https://github.com/mapsme/omim/pull/968

> - `?gpstrackaccuracy:XXX`: Relic from the MapsMe fork and does not work on Organic Maps yet. Changes the accuracy of the gps while recording tracks. Replace `XXX` by the desired horizontal accuracy.

Reviewer 2: ```suggestion
- `?gpstrackaccuracy:XXX`: Changes the accuracy of the GPS while recording tracks. Replace `XXX` by the desired horizontal accuracy. Works now only on iOS.
```
